core/libraries/shortcodes/EE_Recipient_List_Shortcodes.lib.php

```diff
@@ -170,9 +170,9 @@ private function _get_recipient_datetime_list_parsed(EE_Messages_Addressee $data
     {
         // first get registrations just for this attendee.
         $att = $data->att_obj;
-        $registrations_on_attendee = $att instanceof EE_Attendee ? $data->attendees[ $att->ID() ]['reg_objs'] : null;
+        $registrations_on_attendee = $att instanceof EE_Attendee ? $data->attendees[ $att->ID() ]['reg_objs'] : array();
         $registrations_on_attendee = empty($registrations_on_attendee) && $data->reg_obj instanceof EE_Registration
-            ? array($data->reg_obj) : array();
+            ? array($data->reg_obj) : $registrations_on_attendee;
```

Comment:
the fix is perfect but the formatting is a bit off...

should be:

```php
$registrations_on_attendee = empty($registrations_on_attendee) && $data->reg_obj instanceof EE_Registration
	? array($data->reg_obj)
	$registrations_on_attendee;
```

It's only acceptable to have the ternary options on one line if the entire statement fits on one line, like:

```php
$registrations_on_attendee = $att instanceof EE_Attendee ? $data->attendees[ $att->ID() ]['reg_objs'] : array();
```

is POIFECT!!!

Reply:
Ah, thanks Brent.

Fixed.